app/user.go

```diff
@@ -945,6 +945,13 @@ func (a *App) userDeactivated(user *model.User) *model.AppError {
 
 	a.SetStatusOffline(user.Id, false)
 
+	// when disable a user, this userDeactivated is called or the user and the
+	// bots the user owns. Only notify once, when the user is the owner, not the
+	// owners bots.
+	if !user.IsBot {
+		a.notifySysadminsBotDisabled(user.Id)
```

Comment:
Shouldn't this only happen when the bots are actually disabled below?

Reply:
Sorry, the confusion caused here is from the method name, `notifySysadminsBotDisabled()`.  I renamed the function to `notifySysadminsBotOwnerDeactivated()` which is more appropriate.  The purpose of the function is to notify admins that a user, that owned bots, has been deactivated.  The message displayed to the sysadmins will depend on the setting of `*a.Config().ServiceSettings.DisableBotsWhenOwnerIsDeactivated`